pythermalcomfort/models/thi.py

```
@@ -0,0 +1,49 @@
+from typing import Union
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)